frontend/cypress/integration/common/overview.ts

```diff
@@ -0,0 +1,26 @@
+import { And, Given, Then } from "cypress-cucumber-preprocessor/steps";
+
+const url = "/";
+
+/*
+    TODO Prerequisites
+
+    This test assumes that integration test cluster has installed the "errors-rate" demo app
+    https://github.com/kiali/demos/tree/master/error-rates#quick-start
+*/
```

Comment:
just make sure whatever way we choose to install, that it has to support installing on upstream istio and on maistra/openshift (I believe the scripts in hack/ do already support installing on Openshift/maistra)

Reply:
Good comment.
Well, I think we should use the "kiali/demos" for testing.
Those are designed to cover almost all the scenarios that Kiali has features (HTTP, grpc traffic), fixed error rates, all combinations of Gateways/ServiceEntries, runtimes metrics, etc.
If some scenarios are not present, let's update the demos.

I'm not convinced that the UI test should install the demo, I put a comment in the code to make it open or discuss.

I'd expect that the pipeline or a hack/script, if we can avoid the interaction from the cypress in systems tasks, perhaps may help.

But again, this is just a draft for learning and syncing in the same way to write thing, as you can see the tests are very minimal and those can be adapted to any flow.

Reply:
> I'd expect that the pipeline or a hack/script, if we can avoid the interaction from the cypress in systems tasks, perhaps may help.

Yes if we're deploying the demos, it is probably better to setup those outside of the cypress tests since those live in a separate repo.